evap/contributor/templates/contributor_course_form.html

```diff
@@ -17,26 +17,37 @@
     <form method="POST" class="form-horizontal multiselect-form select2form" id="course-form">
         {% csrf_token %}
 
-        {% include 'course_form_general_questions.html' with course=course course_form=form %}
-
-        {% include 'contribution_formset.html' with formset=formset staff=False editable=editable%}
+        <div class="card mb-3">
+            <div class="card-body">
+                {% include 'course_form_general_questions.html' with course=course course_form=form %}
```

Comment:
Are there any rules by which we decide whether a title for a card becomes part of it body or its heading? I find it a little inconsistent. Especially on the contributor and staff course edit page I feel like having some sort of line or other visual guidance that distincts the title from the content could make finding it easier. Compare the two attached screenshots.
![grafik](https://user-images.githubusercontent.com/13838962/34800659-0568e7a6-f665-11e7-8757-f34485ec067e.png)

![grafik](https://user-images.githubusercontent.com/13838962/34800664-0ab69d8e-f665-11e7-90c7-e0be4949d035.png)

Reply:
okay, card titles are also used in other places, where i introduced cards following the current bootstrap 4 default (e.g. user form, settings page, person import, staff index). would you prefer all of them to be unified?

Reply:
I guess unified looks more... unified? But I don't know yet which of the looks I'd prefer.
Since this was not unified before anyways and I only noticed on for the heading "Course Information" on the course edit page, I'll approve this and think about it some more. If I come up with a good idea, I'll create an issue just for that.

Reply:
perfect, thanks. it won't be the last ui update ;)